new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -268,11 +269,25 @@ public static void collectLogs(String namespace, String resultDir) {
    */
   private static void writeToFile(Object obj, String resultDir, String fileName)
       throws IOException {
+    writeToFile(obj, resultDir, fileName, true);
+  }
+
+  /**
+   * Write the YAML representation of object or String to a file in the resultDir.
+   *
+   * @param obj to write to the file as YAML or String
+   * @param resultDir directory in which to write the log file
+   * @param fileName name of the log file
+   * @param asYaml write as yaml or string
+   * @throws IOException when write fails
+   */
+  private static void writeToFile(Object obj, String resultDir, String fileName, boolean asYaml)
+      throws IOException {
     LoggingFacade logger = getLogger();
     logger.info("Generating {0}", Paths.get(resultDir, fileName));
     if (obj != null) {
       Files.write(Paths.get(resultDir, fileName),
-          dump(obj).getBytes(StandardCharsets.UTF_8)
+          (asYaml == true ? dump(obj).getBytes(StandardCharsets.UTF_8) : ((String)obj).getBytes(StandardCharsets.UTF_8))
```

Comment:
redundant check for boolean.

Reply:
can u clarify where is the redundancy and what do u suggest

Reply:
```suggestion
          (asYaml ? dump(obj).getBytes(StandardCharsets.UTF_8) : ((String)obj).getBytes(StandardCharsets.UTF_8))
```